executor/distsql.go

```diff
@@ -934,14 +947,22 @@ func (w *indexWorker) fetchHandles(ctx context.Context, result distsql.SelectRes
 		}
 	}()
 	retTps := w.idxLookup.getRetTpsByHandle()
+	// for sortedSelectResult, add pids in last column
+	if !w.idxLookup.index.Global && len(w.idxLookup.byItems) > 0 {
+		retTps = append(retTps, types.NewFieldType(mysql.TypeLonglong))
+	}
```

Comment:
Can we move this code block into getRetTpsByHandle?

Reply:
Can't do that. Because we use `getRetTpsByHandle` func in two different places. For create chk, we need add pid column, for create select result, we shouldn't do that.